.github/workflows/lms.yml

```diff
@@ -0,0 +1,52 @@
+name: lms-rust
```

Comment:
Noticed this after I approved, would probably just suggest this match the crate name:

```suggestion
name: lms-signature
```

You might also consider naming this file the same as the crate name, i.e. `lms-signature.yml`

Reply:
Then I remembered we omit it with `ed448`. How about just calling it `lms`?